archway-1/chain.json

```diff
@@ -258,6 +271,10 @@
       {
         "address": "grpc-archway.mzonder.com:443",
         "provider": "MZONDER"
+      },
+      {
+        "address": "https://grpc-archway.mms.team:443",
```

Comment:
Please remove the `https://` from the address since it's not used in the protocol